test/server/lib/route-helpers.js

```diff
@@ -1,136 +1,148 @@
 'use strict';
 
-const moment = require('moment');
 const timeshift = require('timeshift');
 const stub = sinon.stub().resolves('finished');
-const { dateChecker, promiseResponder, dashboard, home, homeError } =
-	require('proxyquire')('../../../src/lib/route-helpers', {
-		'./model': stub
-	});
+const {dateChecker, promiseResponder, home, homeError} =
```

Comment:
We usually put spaces around object contents:
```js
const { dateChecker, promiseResponder, home, homeError } =
```
I'll have to get linting going in this project...